frappe/public/js/frappe/form/footer/attachments.js

```diff
@@ -62,14 +62,19 @@ frappe.ui.form.Attachments = Class.extend({
 		}
 
 		var me = this;
+		var lock_icon = repl('<a href="%(file_path)s"><i class="%(icon)s fa-fw text-warning"></i></a> ', { 
```

Comment:
use js templating  instead of repl? (`repl` was just a hack that should be retired!)

```
`<a href="${file_path}"><i class="${icon}....` 
```